website/frontend/styles/AirQommunitiesSection.scss

```diff
@@ -1,110 +1,110 @@
-@import "lib/variables";
+// @import "lib/variables";
 
```

Comment:
Do we intend to use this commented out code in the near future? If no, then better to just remove it to keep our code base clean. @FaithDaka